services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -1242,44 +1241,51 @@ executeProposalAction ::
     Member TinyLog r,
     CallsFed 'Galley "on-conversation-updated",
     CallsFed 'Galley "on-mls-message-sent",
-    CallsFed 'Galley "on-new-remote-conversation",
+    CallsFed 'Galley "on-new-remote-conversation"
+  )
+
+executeProposalAction ::
+  forall r.
+  ( HasProposalActionEffects r,
```

Comment:
👍

Reply:
Why are `CallsFed 'Galley` constraints part of the `HasProposalActionEffects` type alias, but the `CallsFed 'Brig` constraint is not?

Reply:
Because it's not used by the other functions that use the constraint synonym. It doesn't really matter much, one way or the other.